File: OLD_JULIA/src_describe/standard_deviation.jl
#=
    Returns the standard deviation of the array
    Standard deviation = square root of (1/n * sum of each (element - mean)^2)
=#
function standard_deviation(mean, values_array)
    value = 0
    for elem in values_array
        value += (elem - mean)^2
    end
    return ((value / length(values_array))^1/2)
end

File: src_describe/standard_deviation.py
import math
"""
    Goal: Return the standard deviation of the list
"""

def standard_deviation(data, mean, count):
    variance = 0
    for elem in data:
        if elem != "":
            variance += (float(elem) - mean) ** 2
    return round(math.sqrt(variance / (count - 1)), 2)
